experimental/traffic-portal/src/app/api/server.service.ts

```diff
@@ -186,4 +187,33 @@ export class ServerService extends APIService {
 
 		return this.put(`servers/${id}/status`, {offlineReason, status}).toPromise();
 	}
+
+	/**
+	 * Creating new Status.
+	 *
+	 * @param data containes name and description for the status.
+	 * @returns The 'response' property of the TO status response. See TO API docs.
+	 */
+	public async createStatus(data: ResponseStatus): Promise<ResponseStatus> {
+		return this.post<ResponseStatus>("statuses", data).toPromise();
+	}
+
+	/**
+	 * Updates status Details.
+	 *
+	 * @param data containes name and description for the status., unique identifier thereof.
+	 * @param id The Status ID
+	 */
+	public async updateStatusDetail(data: ResponseStatus, id: number): Promise<ResponseStatus | undefined> {
+		return this.put<ResponseStatus>(`statuses/${id}`, data).toPromise();
+	}
+
+	/**
+	 * Deletes an existing Status.
+	 *
+	 * @param id The Status ID
+	 */
+	public async deleteStatus(id: number): Promise<void> {
+		return this.delete(`statuses/${id}`).toPromise();
+	}
```

Comment:
These new methods need to also be added to the testing service at `src/app/api/testing/server.service.ts`